test/comprehensive/host/teststring.cpp

```diff
@@ -82,7 +84,8 @@ oe_result_t ecall_string_no_null_terminator_modified(
     /* Call enclave function */
     if ((_result = oe_call_enclave_function(
              enclave,
-             all_fcn_id_ecall_string_no_null_terminator,
+             &global_id,
+             __all_ecall_name_table[all_fcn_id_ecall_string_no_null_terminator],
```

Comment:
A hash would work to (along with the name). But I see that as an optimization/implementation detail.